composer.json

```diff
@@ -25,28 +25,26 @@
   },
   "autoload": {
     "psr-4": {
-      "Roots\\Sage\\": "app/lib/Sage/"
+      "App\\": "app/"
```

Comment:
PSR-4 doesn't limit the use of namespace in this way. PSR-4 is only for autoloading classes.

If you call a class `App\Sage\Cheese`, then it will look for it in `app/Sage/Cheese.php`, i.e., PSR-4 autoload works as expected.